portfolio/src/main/webapp/education.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Education & Activities</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body bgcolor="AliceBlue"> 
+      <h3 align="center">Education & Activities</h3>
```

Comment:
Nit: try to keep the indentation consistent for the file. Either two or four spaces.

Reply:
My bad, html style guide says we should indent by 2 spaces.